src/python/hooks/tests/test-file-conventions.yaml

```diff
@@ -0,0 +1,17 @@
+# cspell:disable
+---
+# Options: snake_case, kebab-case, camelCase, PascalCase, UPPER_CASE,
+# and * for any case.
```

Comment:
Is there any process that ensures this comment will be updated if the Python script adds a new type?

If not, I suggest we take this out to reduce documentation maintenance overhead per https://github.com/econia-labs/econia-v5/blob/main/CONTRIBUTING.md#general point 2